packages/discovery-provider/ddl/migrations/0102_add_cover_attribution_and_ownership.sql

```diff
@@ -0,0 +1,10 @@
+BEGIN;
+
+ALTER TABLE tracks 
+ADD COLUMN IF NOT EXISTS cover_attribution JSONB,
```

Comment:
I think it would be better to opt for two db columns for this - it could make querying down the line somewhat easier if we need to - but curious why you opted for JSON here? i'm fine w it

Reply:
Was mirroring our implementation of `stream_conditions` and `download_conditions`, how are we currently querying those?

Reply:
`stream_conditions` is a bit of a weird one because it's polymorphic (can be nft gated, follow gated, etc) so we need to just store a JSON object and rehydrate it. Since cover attribution appears to just be a set of static fields, I would agree with Ray that distinct columns make sense.